package.json

```diff
@@ -20,6 +20,7 @@
     "prepare": "npm run build",
     "snyk-test": "snyk test",
     "pkg-binaries-linux": "npx nexe@3.3.7 dist/index.js  -r './dist/**/*.js' -t linux-x64-12.16.2 -o snyk-api-import-linux",
+    "pkg-binaries-alpine": "npx nexe@3.3.7 dist/index.js  -r './dist/**/*.js' -t alpine-x64-12.9.1 -o snyk-api-import-alpine",
```

Comment:
this is not needed here, `pkg-binaries-linux` is only there for specific tests to run